SPECS/security/security.spec

```diff
@@ -0,0 +1,45 @@
+%global debug_package   %{nil}
+
+Summary:        Security package for Mariner to meet all sorts of compliance rules
+Name:           security
+Version:        1.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://aka.ms/mariner
```

Comment:
Can't use this URL since it links to an internal site. Let's point to our own github for now.